### PR TITLE
Update Windows Installer instructions

### PR DESCRIPTION
**Pull request by BlackstoneEngineering**
We now have a windows installer for yotta, updating instructions appropriately. 
Got rid of get_yotta.py and manual install because we really dont want them doing that.

### CODE REVIEW

**Comment by 0xc0170** [comment]
Do we have any guide for users who like to set up manually, if this PR gets accepted?

**Comment by BlackstoneEngineering** [comment]
Nope, there isn't one for Mac or linux, so don't see why its necessary for
windows

File: docs/index.md [hunk 1]
```diff
@@ -219,50 +219,9 @@ You can also try [installing pip from the Pypy registry](https://pip.pypa.io/en/
...
-    and extract it to a directory (for example `C:\ninja`).
- 
- 5. Add the directory you installed Ninja in to [your path](#windows-path).
-
- 6. Install the **[arm-none-eabi-gcc](#windows-cross-compile) cross-compiler** in
-    order to build software to run on embedded devices.
-
- 7. Finally, **open cmd.exe and run `pip install -U yotta`** to install yotta
-    itself.
+ 1. ** Download the latest [yotta windows installer]**(https://github.com/ARMmbed/yotta_windows_installer/releases/latest).
```
Comment (0xc0170) [inline docs/index.md]: The link is not properly rendered because of those ** I assume

**Comment by BlackstoneEngineering** [comment]
Fixed, should work now. Nice catch!

**Comment by autopulated** [comment]
I'm nervous about completely removing the manual installation instructions, especially until the installer is widely tested (despite what you say for Mac and Linux the full list of dependencies is visible to people in the instructions).

**Comment by BlackstoneEngineering** [comment]
Im more nervous about leaving them in and having people have a terrible time trying to install yotta on windows. If you really want the manual way to be there then I'll add it back.

**Comment by autopulated** [comment]
I think that's best for now – we can definitely recommend the installer, list it first, etc.

Also, it seems like https://github.com/ARMmbed/yotta_windows_installer/releases isn't public yet?

**Pull request merged**